templates/android.os.Bundle.cpp

```diff
@@ -6,11 +6,6 @@ void Bundle::__Initialize() { }
 // --------------------------------------------------------
 // Copied from android::os::BaseBundle
 // --------------------------------------------------------
-::jvoid Bundle::Remove(const ::java::lang::String& arg0) const
```

Comment:
@bitter IIRC this was added to workaround changes in latest API versions but can't remember the details - will it work if the function is now moved back in Java?